scripts/nb-tester/test-notebook.py

```diff
@@ -189,8 +186,8 @@ def find_notebooks() -> list[Path]:
 
 
 def cancel_trailing_jobs(start_time: datetime) -> bool:
-    """
-    Cancel any runtime jobs created after `start_time`.
+    """Cancel any runtime jobs created after `start_time`.
+
```

Comment:
Is this from a linter or just personal preference? Personally, I prefer the original formatting but I don't mind too much as long as we pick one and stick to it.

Reply:
Personal preference so it's less verbose. What I cared about was adding a blank line between the first two sentences because Python docstring convention is to have only one sentence summary in the first paragraph. 

I'll revert the change to be consistent with what we had before. Either style is fine.